### PR TITLE
feat(policy): add privacy policy

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

**Review by copiesofcopies — CHANGES_REQUESTED** [review]
This looks good, with one small caveat: there are a number of markdown links where a space appears after the opening "[" of the link tag, instead of before it, which causes the space before the linked text to be underlined on mouseover. Hardly a big deal, but could be fixed easily with a find/replace of "[ " with " [".

File: policies/privacy-policy.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
+---
+title: Privacy Policy
+layout: layouts/policy.njk
+---
+
+Version 1.2, dated 2021-04-16
+
+The Rust Foundation maintains several services related to the Rust Project, including the[ rust-lang.org](https://www.rust-lang.org/),[ crates.io](https://crates.io), and[ docs.rs](https://docs.rs) (the “Services”). This privacy notice explains what we do with personal information.
```
Comment (Mark-Simulacrum) [inline policies/privacy-policy.md:8]:
```suggestion
The Rust Foundation maintains several services related to the Rust Project, including the [rust-lang.org](https://www.rust-lang.org/), [crates.io](https://crates.io), and [docs.rs](https://docs.rs) (the “Services”). This privacy notice explains what we do with personal information.
```

This seems to be a common thread for multiple links fwiw, and it affects rendering - but the rendering of underlining the space is not horrible so feel free to leave as-is. I think it's probably not *too* hard to fix with a regex.

**Comment by ashleygwilliams** [comment]
ah good catch, I hadn't seen the issue with the links, I can fix it quickly

**Review by Mark-Simulacrum — APPROVED** [review]
Looks good to me modulo links, as noted.

File: policies/privacy-policy.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+[rust-lang.org](https://www.rust-lang.org/) is managed by members of the [Core team](https://www.rust-lang.org/governance/teams/core) and the [Community team](https://www.rust-lang.org/governance/teams/community).
+
+### Visitor logs:
+
+When you visit rust-lang.org, we receive your IP address as part of our standard server logs. We store these logs for 1 year.
+
+
+## **crates.io**
+
+[Crates.io](https://crates.io) is managed by members of the[ Core team](https://www.rust-lang.org/governance/teams/crates-io) and the [Crates.io team](https://www.rust-lang.org/governance/teams/crates-io).
```
Comment (Mark-Simulacrum) [inline policies/privacy-policy.md:22]: Looks like Core here is still wrong?

**Pull request merged**